cardano-node/src/Cardano/Chairman.hs

```diff
@@ -89,7 +95,7 @@ runChairman :: forall blk.
 runChairman ptcl nids securityParam maxBlockNo socketDir tracer = do
 
     (chainsVar :: ChainsVar IO blk) <- newTVarM
-      (Map.fromList $ map (\coreNodeId -> (coreNodeId, AF.Empty Block.GenesisPoint)) nids)
+      (Map.fromList $ map (\coreNodeId -> (coreNodeId, AF.Empty undefined)) nids)
```

Comment:
@mrBliss could you help me filling the `undefined :: Header blk` here.  It is supposed to be the genesis header.

Reply:
Use `AF.Empty AF.AnchorGenesis`